MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "aspect.integrateddynamics.read.integer.redstone.value": "紅石訊號強度",
+    "aspect.integrateddynamics.read.integer.redstone.value.info": "得到確切的紅石訊號強度值",
+    "aspect.integrateddynamics.read.integer.redstone.comparator": "比較器",
+    "aspect.integrateddynamics.read.integer.redstone.comparator.info": "得到比較器產生的紅石訊號強度",
+
+    "aspect.integrateddynamics.write.boolean.redstone": "紅石訊號",
+    "aspect.integrateddynamics.write.boolean.redstone.info": "當布林值為 true 時輸出滿強度的紅石訊號",
+    "aspect.integrateddynamics.write.boolean.redstone.pulse": "脈衝紅石訊號",
+    "aspect.integrateddynamics.write.boolean.redstone.pulse.info": "在給定的值變動時輸出一個特定強度的脈衝",
+
+    "aspect.integrateddynamics.write.integer.redstone": "紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.info": "輸出給定強度的紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.pulse": "脈衝紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.pulse.info": "在給定的值變動時輸出一個特定強度的脈衝",
+
+    "_comment": "# World aspects",
+    "aspect.integrateddynamics.read.boolean.world.entity": "是否有實體",
+    "aspect.integrateddynamics.read.boolean.world.entity.info": "檢查目標空間是否有實體",
+    "aspect.integrateddynamics.read.boolean.world.mob": "是否有生物",
+    "aspect.integrateddynamics.read.boolean.world.mob.info": "檢查目標空間是否有生物",
+    "aspect.integrateddynamics.read.boolean.world.animal": "是否有動物",
+    "aspect.integrateddynamics.read.boolean.world.animal.info": "檢查目標空間是否有動物",
+    "aspect.integrateddynamics.read.boolean.world.player": "是否有玩家",
+    "aspect.integrateddynamics.read.boolean.world.player.info": "檢查目標空間是否有玩家",
+    "aspect.integrateddynamics.read.boolean.world.item": "是否有物品",
+    "aspect.integrateddynamics.read.boolean.world.item.info": "檢查目標空間是否有物品",
+    "aspect.integrateddynamics.read.boolean.world.weather.clear": "是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.clear.info": "檢查天氣是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.raining": "是否為雨天",
+    "aspect.integrateddynamics.read.boolean.world.weather.raining.info": "檢查天氣是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.thunder": "是否為雷雨",
+    "aspect.integrateddynamics.read.boolean.world.weather.thunder.info": "檢查天氣是否為雷雨",
+    "aspect.integrateddynamics.read.boolean.world.isday": "是否為白天",
+    "aspect.integrateddynamics.read.boolean.world.isday.info": "檢查時間是否為白天",
+    "aspect.integrateddynamics.read.boolean.world.isnight": "是否為晚上",
+    "aspect.integrateddynamics.read.boolean.world.isnight.info": "檢查時間是否為晚上",
+
+    "aspect.integrateddynamics.read.integer.world.entity": "實體數量",
+    "aspect.integrateddynamics.read.integer.world.entity.info": "目標空間的實體數量",
+    "aspect.integrateddynamics.read.integer.world.raincountdown": "雨天冷卻",
+    "aspect.integrateddynamics.read.integer.world.raincountdown.info": "此世界在幾 tick 後會下雨",
+    "aspect.integrateddynamics.read.integer.world.ticktime": "刻時間",
+    "aspect.integrateddynamics.read.integer.world.ticktime.info": "此世界平均每 tick 的時間（ms）",
```

參考**frametime**翻譯，加入「此數值為計算每一刻所耗費之時間」的概念
考量其他地方也可能出現單位**ms**，故保持英文以確保統一性
```suggestion
    "aspect.integrateddynamics.read.integer.world.ticktime.info": "此世界平均每刻耗時（ms）",
```